cylc/flow/cfgspec/globalcfg.py

```diff
@@ -457,6 +469,40 @@
                         install target = localhost
             ''')
 
+            Conf('clean job submission environment', VDR.V_BOOLEAN, False,
+                 desc='''
+                Job submission subprocesses inherit their parent environment by
+                default. So remote job submissions inherit the default
+                non-interactive shell environment, but local ones inherit the
+                scheduler environment. This means local jobs see the scheduler
+                environment unless the local batch system prevents it, which
+                can cause problems - e.g. scheduler ``$PYTHON...`` variables
+                can affect Python job applications). For consistent handling of
+                local and remote jobs a clean job submission environment is
+                recommended, but it is not the default because it prevents
+                local task jobs from running unless the ``cylc`` version
+                selection wrapper script is installed in ``$PATH`` (a clean
```

Comment:
Python job script!

Reply:
Ah, what's your point?

Reply:
Are you referring to 
> can affect Python job applications

I should change that to "Python programs executed in task job scripts"

Reply:
(Done)